contrib/grain_size/phase_distribution_function.ipynb

```diff
@@ -0,0 +1,145 @@
+{
```

Comment:
I also just went through all equations for our discussion with Elvira, and I would definitely keep the notebook to explain where the equations are coming from.

Reply:
Got it, thanks!